tests/robottelo/test_cli.py

```diff
@@ -1,8 +1,13 @@
-import mock
+import six
 import unittest2
 
 from robottelo.cli.base import Base
 
+if six.PY2:
+    import mock
+else:
+    from unittest import mock
```

Comment:
Conditional dependencies cannot be added in `setup.py`. Instead, they must be added in a requirements file. See [here](https://github.com/SatelliteQE/nailgun/blob/master/requirements-dev.txt#L3) for an example.